src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Yes. Whether `AkkaMaterializerSpec` is needed is independent of whether `UnitSpec` is used.

`AkkaMaterializerSpec`'s job is to provide an implicit Materializer. An implicit Materializer is needed by `bodyOf` (whichever version of `bodyOf` you use - `TestWithoutPlayAppSpec.bodyOf` and `UnitSpec.bodyOf` both need one).

The reason I didn't use `UnitSpec` is that the other specs in this library don't use it and it's not on the classpath. In doing so I actually discovered that I like the traits in `play.api.test` better than UnitSpec because they are more composable. However I don't mind adding a dependency on `hmrctest` and making `TestWithoutPlayAppSpec` use `UnitSpec` if that's what you prefer - but that won't remove the need for `AkkaMaterializerSpec`.

The only way I can think of of getting rid of `AkkaMaterializerSpec` is to merge it into `TestWithoutPlayAppSpec` (which is the only test that needs it so far).

Aside: `UnitSpec.bodyOf` has only needed a Materializer since the Play 2.5 upgrade.